app/controllers/users_controller.rb

```diff
@@ -68,6 +68,16 @@ def destroy
     redirect_to users_url, notice: exception.message
   end
 
+  def orders
+    @user = User.find(session[:user_id])
+    @orders = @user.orders
+  end
+
+  def line_items
+    @user = User.find(session[:user_id])
```

Comment:
Always handle failed cases also. If session[:user_id] is not found, your endpoint will give 500.
Endpoint should return proper message with proper status code.